flexmeasures/ui/crud/assets.py

```diff
@@ -195,6 +195,23 @@ class AssetCrudUI(FlaskView):
 
     route_base = "/assets"
 
+    def get_breadcrumb_info(self, asset: GenericAsset):
+        account_id = asset.account_id
+        account = Account.query.get(account_id)
+        if account is None:
+            account_id = None
+        account_name = (
+            Account.query.get(account_id).name if account_id is not None else None
+        )
+        account_url = (
+            url_for("AccountCrudUI:get", account_id=account_id)
+            if account_id is not None
+            else None
+        )
+        parent_asset = GenericAsset.query.get(asset.id).parent_asset
```

Comment:
`asset.parent_asset`?

Reply:
The `parent_asset` field wasn't being loaded before and that's why I had to add this line. I've modified the piece of code where we are building the object from the HTTP request.